### PR TITLE
Implement fixture.before and fixture.after hooks (closes #903)

### LINKED ISSUE #903: before & after hooks for a fixture
### Are you requesting a feature or reporting a bug?
Requesting a feature.

### What is the current behavior?
There is no ability to run some setup/tear down code before/after all tests in a fixture.

### What is the expected behavior?
To have such the ability.

#### Provide the test code and the tested page URL (if applicable)
Test code

```js
fixture('My cool fixture')
    .before(() => {
        // do something that needs to be done once _before_ all tests
    })
    .after(() => {
        // do something that needs to be done once _after_ all tests
    })
```

### PR DESCRIPTION
**Pull request by inikulin**
\cc @AlexanderMoskovkin @georgiy-abbasov @AndreyBelym

### CODE REVIEW

File: src/.eslintrc [hunk 1]
```diff
@@ -6,5 +6,8 @@
   },
   "plugins": [
     "babel"
-  ]
+  ],
+  "globals": {
```
Comment (AndreyBelym) [inline src/.eslintrc]: Maybe use 
```js
"env": {
    "es6": true
}
```
instead?

Comment (inikulin) [inline src/.eslintrc]: No, we've decided to use pinkie promise, so enabling `es6` will not report about `Promise` global

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Pull request merged**